dist/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Todo List</title>
+    <!-- <link rel="stylesheet" href="style.css"> -->
+</head>
+
+<body>
+    <div class="title">
+
+        <h1>Todo List App</h1>
+
+    </div>
+    <div class="grid-container">
+
+        <div class="left-side items">
+            <!-- your lists -->
+            <h2>Your Lists</h2>
+            <div class="list-info">
+                <div id="link-list"></div>
+                <!-- <a href="#" id="listName">My Travel list</a> -->
+            </div>
+            <div class="newBtn">
+                <button class="#" id="addNewList">Add a new list</button>
+            </div>
+        </div>
+        <!-- Show the content -->
+        <div class="middle-side items">
+            <!-- <div class="listName">My Travel list</div> -->
+            <!-- checkbox, todo's title,due date prioprity and button that shows and hides the description edit and delete todo -->
+            <div id="todo-box"></div>
+            <div class="newTodoBtn">
+                <button class="#" id="addNewTodo">Add a new todo</button>
+            </div>
+            <div class="new-todo">
+                <form action="#" id="todo-form">
+                    <label for="title">Title</label>
+                    <div>
+                        <input type="text" name="title" id="title" required>
+                    </div>
+
+                    <label for="description">Description</label>
+                    <div>
+                        <input type="text" name="description" id="description">
+                    </div>
+                    <label for="dateDue">Date Due</label>
+                    <div>
+                        <input type="date" name="dateDue" id="dateDue">
+                    </div>
+                    <label for="priority">Priority</label>
+                    <select name="priority" id="priority">
+                        <option value="low">Low</option>
+                        <option value="medium">Medium</option>
+                        <option value="high">High</option>
+                    </select>
+                    <div class="cta-btns">
+                        <button class="save-btn">Save</button>
+                        <button class="cancel-btn">Cancel</button>
+
```

Comment:
Opps, User should not be able to create an empty `todo`;
![image](https://user-images.githubusercontent.com/31897434/84423430-5bf0ff00-ac16-11ea-8c89-0de8c1fe4ce8.png)